app/_components/commodities/Form.tsx

```diff
@@ -113,109 +114,136 @@ const Form: React.FC<FormProps> = ({ onSubmitHandler, isLoading }) => {
+              </CheckboxGroup>
             </Stack>
-          </RadioGroup>
-        </FormControl>
-        <FormControl isInvalid={!!errors.minSupply || !!errors.minDemand}>
-          <FormLabel>I am looking to:</FormLabel>
-          <Stack spacing={8} direction="row" margin={8} flexWrap="wrap">
-            <ButtonGroup size="md" isAttached>
+          </FormControl>
+          <FormControl
+            width="100%"
+            isInvalid={
+              !!(errors.maxLandingPadSize && errors.maxLandingPadSize.message)
+            }
+          >
+            <FormLabel>Landing Pad Size</FormLabel>
+            <RadioGroup>
+              <Stack
+                borderWidth="1px"
+                borderRadius="md"
+                borderColor={selectColor(isDark, 'border')}
+                padding={3}
+                spacing={5}
+                direction="row"
+              >
+                {['Small', 'Medium', 'Large'].map((value, index) => (
+                  <Radio
+                    colorScheme="orange"
+                    key={index}
+                    value={value.toLowerCase()}
+                    borderColor={selectColor(isDark, 'border')}
+                    {...register('maxLandingPadSize', {
+                      required: true,
+                    })}
+                  >
+                    {value}
+                  </Radio>
+                ))}
+                <FormErrorMessage marginY="auto">
+                  {errors.maxLandingPadSize && 'Select a landing pad size'}
+                </FormErrorMessage>
+              </Stack>
+            </RadioGroup>
+          </FormControl>
+        </Stack>
+        <Stack
+          direction={{ base: 'column', sm: 'column', md: 'row', lg: 'row' }}
+          spacing={4}
+          width="100%"
+          marginTop={4}
+        >
+          <FormControl
+            width="100%"
+            isInvalid={!!errors.minSupply || !!errors.minDemand}
+          >
+            <FormLabel>I am looking to:</FormLabel>
+            <ButtonGroup
+              isAttached
+              borderColor={selectColor(isDark, 'border')}
+              variant="outline"
+            >
               <Button
-                onClick={() => setIsBuying(true)}
                 variant={isBuying ? 'customButton' : 'outline'}
-                leftIcon={<CheckIcon opacity={isBuying ? 1 : 0} />}
-                width={[24, 36]}
-                style={{ marginInlineEnd: 0 }}
+                onClick={() => setIsBuying(true)}
+                leftIcon={<CheckIcon />}
               >
                 Buy
               </Button>
               <Button
+                variant={isBuying ? 'outline' : 'customButton'}
                 onClick={() => setIsBuying(false)}
-                variant={!isBuying ? 'customButton' : 'outline'}
-                leftIcon={<CheckIcon opacity={!isBuying ? 1 : 0} />}
-                width={[24, 36]}
+                leftIcon={<CheckIcon />}
```

Comment:
FYI: the original intent behind the check was to clarify which button was selected. I.e. is the colored one selected or the uncolored one? The check icon confirms which one (trying to follow Material Design docs). I used opacity to hide the unselected icon so that there wasn't lateral shift as you toggle. 

I don't have a major issue with this change, but want to clarify the goal of the original implemenation.

Reply:
No you are right, I must have messed with that behavior. If you can pull and fix, great, otherwise I’ll take a look tomorrow. Thanks

Reply:
Fixed it! I also had it set `display` instead of `opacity`, I believe this was discussed in a voice call at some point.

Reply:
I personally think that it doesn't need the checkbox. It messes up with the symmetry of the button widths and gives a jarring effect on selecting one or the other. Without checkbox also is obvious enough
![Screen Shot 2023-07-29 at 4 19 17 PM](https://github.com/ed-pilots-network/frontend/assets/1340144/93e62f8b-2328-4ecf-b066-6d266282e3ac)
![Screen Shot 2023-07-29 at 4 19 35 PM](https://github.com/ed-pilots-network/frontend/assets/1340144/879fcffb-7c2f-4c61-81e9-a3d6fd1a579b)

Reply:
Removing the check icon for now. We can revisit this discussion if needed.